site/documentation/content/user-guide/device-registry.md

```diff
@@ -299,9 +299,9 @@ HTTP/1.1 204 No Content
 ETag: becc93d7-ab0f-48ec-ad26-debdf339cbf4x
 ~~~
     
-This uses a convenient option which lets the Device Registry do the hashing of the password. The following command retrieves the credentials that are stored by the Device Registry as a result of the command above: 
-    
-~~~sh    
+This lets the Device Registry do the hashing of the password. The following command retrieves some metadata credentials that are stored by the Device Registry as a result of the command above: 
```

Comment:
I would rather keep the original wording and instead explain explicitly, why the response payload does not contain the password itself ...

Reply:
I updated the wording. I am not really happy with it though, I am open to suggestions :)